recipes/nmslib/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
Are the `bld.bat` and `build.sh` necessary, or could they be replaced with:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35

Reply:
Yes it needs different commands for Linux and windows

Reply:
Could you elaborate on why there is a difference? Is it possible to use `PIP_NO_CACHE_DIR=True $PYTHON -m pip install --no-deps --no-binary :all: ` for both linux and windows? I think we should be using pip for both, unless there is a good reason not to. You can also replace the two separate script files with:

```
script: ...  # [win]
script: ...  # [unix]
```
 to just do everything in the `meta.yaml`.

Reply:
OK I didn't realize that `PIP_NO_CACHE_DIR=True` syntax would be fine on windows - not familiar with windows scripts. But now this builds on windows with everything in `meta.yaml`. Thanks! 

I do still see one issue though - I saw the following warning in the log

```
Your CPU supports instructions that this binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2
For maximum performance, you can install NMSLIB from sources 
pip install --no-binary :all: nmslib
import: 'nmslib'
```
- even though I did use the command suggested and I believe it is building from source. Do you have any idea about why?